### PR TITLE
Completed adding support for Activity api

### PR DESCRIPTION
**Pull request by AQI-DanG**

### CODE REVIEW

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-	  <VersionPrefix>11.7.2</VersionPrefix>
+	  <VersionPrefix>11.8.0</VersionPrefix>
 	  <!--For Feature packages update the VersionSuffix with the Jira ticket number-->
-	  <VersionSuffix>Alpha1</VersionSuffix>
+	  <VersionSuffix>Alpha-2</VersionSuffix>
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj:8]: seems weird to increment the version number and pre-release version

File: src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs [hunk 2]
```diff
@@ -4,164 +4,186 @@
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using System.Linq;
+using System.Net;
+using System.Text;
 using ONE.Models.CSharp;
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs:9]: I think the usings on line 9 and 10 could be combined.  I'd probably suggest using Proto. everywhere in this file.

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj (hunk 1 above)
Comment (AQI-DanG) [reply src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj:8]: Currently this package is only available on my local machine. My plan was to remove the pre-release suffix after code review.

Comment (AQI-DanG) [reply src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj]: Removed the suffix

File: src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs (hunk 2 above)
Comment (AQI-DanG) [reply src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs]: Combined the using statements

**Review by joezhou-aqi — APPROVED** [review]

**Pull request merged**